packages/react-components/components/SectionHeadNew.tsx

```diff
@@ -0,0 +1,49 @@
+import ItemSeparator from '@celo/react-components/components/ItemSeparator'
```

Comment:
Added a new component so other screens using `<SectionHead />` are not impacted by the new designs. 
The new design doesn't work well on some existing screens.